### PR TITLE
refactor(avform): add spinBox for custom FPS

### PR DESCRIPTION
**Pull request by Pik-9**
Sometimes the auto detected FPS rate causes trouble. So the user should have the choice to intervene.

---

This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/3215)

### CODE REVIEW

**Comment by sudden6** [comment]
I see multiple issues with this PR:
- Setting arbitrary fps does neither work, nor does it make sense. We only have a list of supported resolutions and associated fps we can choose from.
- your change doesn't update the video preview according the fps settings. How should the user know if his setting actually works?
- some style issues, see the other comments

File: src/widget/form/settings/avform.cpp [hunk 1]
```diff
@@ -54,6 +54,7 @@ AVForm::AVForm() :
     connect(bodyUI->outDevCombobox, qcbxIndexChangedStr, this, &AVForm::onOutDevChanged);
     connect(bodyUI->videoDevCombobox, qcbxIndexChangedInt, this, &AVForm::onVideoDevChanged);
     connect(bodyUI->videoModescomboBox, qcbxIndexChangedInt, this, &AVForm::onVideoModesIndexChanged);
+    connect(bodyUI->FPSSpinbox, SIGNAL (valueChanged (int)), this, SLOT (onVideoFPSChanged (int)));
```
Comment (sudden6) [inline src/widget/form/settings/avform.cpp:57]: We try to use the new connect syntax from http://doc.qt.io/qt-5/signalsandslots.html

File: src/widget/form/settings/avform.cpp [hunk 2]
```diff
@@ -463,4 +475,4 @@ void AVForm::retranslateUi()
 void AVForm::on_btnPlayTestSound_clicked(bool checked)
 {
     mPlayTestSound = checked;
-}
+}
```
Comment (sudden6) [inline src/widget/form/settings/avform.cpp:478]: Seems like your editor does something with the end of the file

Comment (antis81) [inline src/widget/form/settings/avform.cpp:478]: Git indicates a missing newline `\n` character here. Please restore that line.

File: src/widget/form/settings/avform.h [hunk 3]
```diff
@@ -85,4 +86,4 @@ private slots:
     QVector<VideoMode> videoModes;
 };
 
-#endif
+#endif
```
Comment (antis81) [inline src/widget/form/settings/avform.h:89]: Same here -> please restore.

**Comment by zetok** [comment]
Hi,

As you might be aware, repo has been moved under `qTox` org on github.

What does it have to do with your PR, you might ask.

Well, it doesn't change all that much, except that workflow has been slightly improved – now there are required checks that PR needs to pass in order to be merged.

Those checks are:
- travis (checks whether PR builds & commit message format)
- reviewable (requirement for PR to be reviewed & accepted)

Since osx travis was broken for a bit, you will need to rebase your PR on newest master to get it to pass travis check. Sorry for inconvenience.

Reminder: as always, PR needs to be possible to merge without conflicts. Make sure it does ;)

**Comment by Pik-9** [comment]
This pull request seems to be obsolete, now!
I'll close it. Thanks for your reminder anyway! ;-)

**Pull request closed without merging**